clang/lib/Driver/ToolChains/Darwin.h

```diff
@@ -223,6 +223,11 @@ class LLVM_LIBRARY_VISIBILITY MachO : public ToolChain {
     // There aren't any profiling libs for embedded targets currently.
   }
 
+  // Return the full path of the compiler-rt library on a Darwin MachO system.
+  // Those are under <resourcedir>/lib/darwin/<...>(.dylib|.a).
+  std::string getCompilerRT(const llvm::opt::ArgList &Args, StringRef Component,
```

Comment:
What is the difference between the MachO `getCompilerRT` and the Darwin one? Probably worth having a sentence or two to make it clear what the difference is!

Reply:
Whoops, this was a copy and paste error inadvertently repeating the same comment twice. Now it should have the correct path in the comment. Do you think we should explain other differences here (apart from the path where the libraries can be found)? If that is the case, could you point me to some resources about non-Darwin MachO targets?